tests/unit/mock_data.py

```diff
@@ -207,7 +207,7 @@ def provisioning_config(inventory_layout=None):
             },
             "pubkey": "config/id_rsa.pub",
             "reserve_duration": 86400,
-            "max_attempts": 240,
+            "ks_meta": "GLOBAL_KS_META",
```

Comment:
Wouldn't `ks_meta` be different per os? I.e. shouldn't we rather have it as a dict with OS keys and with `default` as a fallback?

Reply:
I can rewrite it